src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,109 @@
+package pages;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+    WebDriver driver;
+    WebDriverWait wait;
+
+    //WebElements
+    @FindBy(css = "img.avatar")
+    WebElement userAvatarIcon;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example


```
    @FindBy(css = "img.avatar")
    private WebElement userAvatarIcon;
```